crates/module-system/sov-modules-api/src/state/accessors/tests.rs

```diff
@@ -151,7 +153,7 @@ fn test_values<S: Spec>(state: &mut StateCheckpoint<S>) {
         let is_cached = state.is_value_cached(NAMESPACE, &state_value.slot_key());
         assert_eq!(is_cached, IsValueCached::Yes(AccessSize::Read(8)));
 
-        let maybe_size = state.get_size(NAMESPACE, &state_value.slot_key());
+        let maybe_size = state.get_size(NAMESPACE, &state_value.slot_key(), &mut metric);
```

Comment:
It is out of the scope of this test, but maybe we can make an exception and check that metrics are tracker properly?